crates/hdx_proc_macro/src/def.rs

```diff
@@ -1350,8 +1350,8 @@ impl DefType {
 
 	pub fn requires_allocator_lifetime(&self) -> bool {
 		if let Self::Custom(DefIdent(ident), _) = self {
-			return matches!(ident, &atom!("OutlineColor") | &atom!("BorderTopColor") | &atom!("AnchorName"));
-		}
+			return matches!(ident, &atom!("OutlineColor") | &atom!("BorderTopColor") | &atom!("AnchorName") | &atom!("DynamicRangeLimitMix"));
```

Comment:
Because `DynamicRangeLimitMix` required the `<'a>` lifetime, it needs to be added here. This is... not ideal, but it works for now. The generator needs to be smarter.